src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -14,18 +15,20 @@ import useViewTracker from '#hooks/useViewTracker';
 import useOptimizelyVariation from '#hooks/useOptimizelyVariation';
 import useToggle from '#hooks/useToggle';
 
-const eventTrackingData = {
+const getEventTrackingData = optimizely => ({
   block: {
     componentName: 'features',
+    optimizely,
   },
-};
+});
 
 const HIGH_IMPACT_EXPERIMENT_ID = 'high_impact_feature_analysis_promo';
 const HIGH_IMPACT_VARIATION = 'variation_1';
 
 const PromoListComponent = ({ promoItems, dir }) => {
   const { serviceDatetimeLocale } = useContext(ServiceContext);
-  const viewRef = useViewTracker(eventTrackingData.block);
+  const { optimizely } = useContext(OptimizelyContext);
```

Comment:
would this be maybe be better in the EventTrackingContext? that way you wouldn't need to write this line and pass to the hooks every time you wanted to use it

Reply:
Originally it was in the [hooks](https://github.com/bbc/simorgh/blob/8c9e7f7fec7fd280d0f3abd24f9fd0cea7db76c3/src/app/hooks/useViewTracker/index.jsx#L37), but after building Simorgh and analysing the asset chunks, optimizely was being included in other page type bundles, this was due to MostRead and EpisodeList making use of click and view tracking. So I thought a solution, for now, would be to pass the optimizely object instead to stop the bundles being polluted. Unless there's a way to dynamically import the library in the view and click tracking hooks?